aep/0140.yaml

```
@@ -0,0 +1,17 @@
+rules:
+  aep-140-boolean-field-naming:
+    description: 'Boolean fields should omit the prefix "is".'
+    severity: warn
+    given: '$..[?(@.type == "boolean" && (@property.match(/^is/)))]~'
```

I don't think you need to check the property name here _and_ in the pattern function:
```suggestion
    given: '$..[?(@.type == "boolean")]~'
```